spanner/api/SpannerTest/Tests.cs

```diff
@@ -133,13 +128,83 @@ public SpannerTests(SpannerFixture fixture)
                 if (!_fixture.s_initializedDatabase)
                 {
                     _fixture.s_initializedDatabase = true;
+                    DeleteStaleBackupsAndDatabases();
                     InitializeInstance();
                     InitializeDatabase();
                     InitializeBackup();
                 }
             }
         }
 
+        void DeleteStaleBackupsAndDatabases()
+        {
+            DatabaseAdminClient databaseAdminClient = DatabaseAdminClient.Create();
+            var instanceName = InstanceName.FromProjectInstance(_fixture.ProjectId, _fixture.InstanceId);
+            var databases = databaseAdminClient.ListDatabases(instanceName)
+                .Where(c => c.DatabaseName.DatabaseId.StartsWith("my-db-") || c.DatabaseName.DatabaseId.StartsWith("my-restore-db-"));
+            var databasesToDelete = new List<string>();
+
+            // Delete all the databases created before 4 hrs.
+            var timestamp = DateTimeOffset.UtcNow.AddHours(-4).ToUnixTimeSeconds();
+            foreach (var database in databases)

```

Comment:
Ain't it easier to use the database creation time here? It's fine to use a timestamp for the name, but parsing that back to get the creation time seems a little too much.

Reply:
I was thinking on same but [In proto](https://github.com/googleapis/googleapis/blob/cd804bab06e46dd1a4f16c32155fd3cddb931b52/google/spanner/admin/database/v1/spanner_database_admin.proto#L364-L365) it says that the create_time might not be available every time(If exists).